packages/playground/src/components/manage_gateway_dialog.vue

```diff
@@ -245,7 +251,7 @@ export default {
     const selectedIPAddress = ref<VMNetwork | null>(null);
     const networkName = props.vm
       ? (props.vm.interfaces[0].network as string)
-      : (props.k8s.masters[0].interfaces[0].network as string);
+      : (props.k8s!.masters[0].interfaces[0].network as string);
```

Comment:
again use `?` as we are not sure if `props.k8s` here or not

Reply:
Resolved